src/clr-angular/forms/combobox/_properties.combobox.scss

```diff
@@ -7,6 +7,11 @@
     :root {
       --clr-ng-select-trigger-color: var(--clr-color-neutral-500);
       --clr-ng-select-font-size: #{$clr-p2-font-size};
+      --clr-ng-select-input-background: #{$clr-ng-select-input-background-color};
```

Comment:
Update prefixes throughout to be `--clr-combobox` and not `--clr-ng-select`

Reply:
Done